### PR TITLE
Add readydata to more integration tests

### PR DESCRIPTION
**Pull request by fassadlr**

### CODE REVIEW

File: src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs [hunk 1]
```diff
@@ -205,12 +206,10 @@ public void Given_TheNodeHadAReorg_And_WalletTipIsBehindConsensusTip_When_ANewBl
         {
             using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).WithWallet().Start();
```
Comment (dangershony) [inline src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs]: Does WithReadyBlockchainData replace WithWallet?

Comment (fassadlr) [reply src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs]: Yes, the wallet is included in the ReadyData 👍 So no need to create it in the test.

Comment (zeptin) [reply src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs]: @dangershony there is a wallet file inside the data zips, so I believe it does.

File: src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs [hunk 2]
```diff
@@ -249,11 +248,9 @@ public void Given_TheNodeHadAReorg_And_ConensusTipIsdifferentFromWalletTip_When_
         {
             using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).WithDummyWallet().Start();
+                CoreNode stratisSender = builder.CreateStratisPowNode(this.network).WithReadyBlockchainData(ReadyBlockchain.BitcoinRegTest10Miner).Start();
                 CoreNode stratisReceiver = builder.CreateStratisPowNode(this.network).Start();
-                CoreNode stratisReorg = builder.CreateStratisPowNode(this.network).WithDummyWallet().Start();
```
[the remark this answers is not in the document]
Comment (fassadlr) [reply src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTests.cs]: Fixed 👍

**Pull request merged**